[0m[[0m[31merror[0m] [0m[0mC:\Users\Federico\Exercism\scala\etl\src\main\scala\Etl.scala:7:36: not found: value in[0m
[0m[[0m[31merror[0m] [0m[0m    def transform(a : In) : Out =  in.flatMap(x => x._2.map(u => (u, x._1)))[0m
[0m[[0m[31merror[0m] [0m[0m                                   ^[0m
[0m[[0m[31merror[0m] [0m[0mone error found[0m
